configuration/export.go

```diff
@@ -11,11 +11,28 @@ package configuration
 type Exporter struct {
 	// Addr specifies address where exporter server starts
 	Addr string
+	Auth Auth
+}
+
+// Auth specifies parameters for a token-based authorization of an observer
+type Auth struct {
```

Comment:
annotate it like this pls: https://github.com/insolar/block-explorer/blob/master/configuration/configuration.go

Reply:
@Vany The Insolar platform code does not use the "YamlTemplater" from the insconfig library. It will be useless work to cover an exporter config with an insconfig annotations.
I think this work should be done in a separate task for the whole configuration and adding the YamlTemplater.